_layouts/layout.html

```diff
@@ -150,7 +150,7 @@
     <section class="sidebar">
 
       <a href="/" class="name">
-        <span>RPK</span>
+        <span></span>
```

Comment:
I don't really like empty spans but color: transparent and font-size: 0 is a bad tactic seo-wise. I think there is no need to use them here. We could use the text-indent: 100% or replace it with inline svg (even better).